packages/account/src/BiconomySmartAccount.ts

```diff
@@ -157,18 +160,18 @@ export class BiconomySmartAccount extends SmartAccount implements IBiconomySmart
     } else {
       callData = this.getExecuteBatchCallData(to, value, data)
     }
-    const address = await this.getSmartAccountAddress()
-    this.proxy = await SmartAccount_v100__factory.connect(address, this.provider)
-    const nonce = await this.isAccountDeployed(address) ? await this.nonce() : BigNumber.from(0)
-    
+    let nonce = BigNumber.from(0)
+    try {
+      nonce = await this.nonce()
+    } catch (error) {
+    }
```

Comment:
Write a comment in catch block, mentioning 
// Not throwing this error as nonce would be 0 if this.nonce() throw exception, which is expected flow.

Reply:
resolved